rir/src/compiler/translations/pir_2_rir.cpp

```diff
@@ -93,13 +93,13 @@ class SSAAllocator {
 
     SSAAllocator(Code* code, bool verbose)
         : cfg(code), dom(code), code(code), bbsSize(code->nextBBId) {
-        computeLiveness(verbose);
+        computeLiveness(verbose & 0X10000);
```

Comment:
Nitpick: I prefer to write my hexadecimal literals with a lowercase `x`. Not sure what the consensus here is.

More generally: it would be nice to use names (e.g. `const unsigned int flagName = ...`, or an enum) rather than magic constants.

Reply:
Missed replacing this magic constant.